### PR TITLE
Add HEMTT setup script for Windows and document HEMTT use

### PR DESCRIPTION
**Pull request by jonpas**
**When merged this pull request will:**
- Add setup script that installs a specific version of HEMTT to the local project
  - Works in combination with `build.bat` in root.
- Document HEMTT setup and builds.

### CODE REVIEW

File: docs/wiki/development/setting-up-the-development-environment.md [hunk 1]
```diff
@@ -38,7 +44,41 @@ To actually get the ACE3 source code on your machine, it is recommended that you
...
+
+### Initial Setup
+
+Execute `tools\setup.bat` (double-click on Windows) or download [HEMTT](https://github.com/BrettMayson/HEMTT/releases/latest) and place the executable in project root (Windows and Linux - binarization supported only on Windows at this time).
+
+#### File Patching Setup
+
+**P-drive is not required for HEMTT or file patching!**
+
+Create the following link. First, create a folder called `z` in your Arma 3 directory. Then run the following command as admin, replacing the text in brackets with the appropriate paths:
```
Comment (BrettMayson) [inline docs/wiki/development/setting-up-the-development-environment.md:59]: This is done automatically with `hemtt launch`

**Review by BrettMayson — APPROVED** [review]

**Comment by jonpas** [comment]
`setup.bat` no longer supports Windows 8. We can re-add that if necessary, but most people are now using Windows 10 and it makes the script shorter and faster.

File: tools/setup.bat [hunk 2]
```diff
@@ -0,0 +1,16 @@
+;@Findstr -bv ;@F "%~f0" | powershell -Command - & pause & goto:eof
```
Comment (PabstMirror) [inline tools/setup.bat:1]: rename to setupHemtt.bat?

Comment (jonpas) [reply tools/setup.bat:1]: Pressed the merge button just as you posted this.

I don't see a reason to rename though, it's documented now and HEMTT should become primary build tool anyways.

**Pull request merged**